INITIAL_SETUP.md

```diff
@@ -0,0 +1,204 @@
+# Instructions for Initial Setup
+
```

Comment:
Might be nice to date the last edit of the page for the non-github savvy

Reply:
I agree it would be nice, but since the reader has to modify files in github anyway, and forgetting to update the date seems likely, will leave out for now.